spec/lib/dispatch-rider/queue_services/file_system_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   describe "#assign_storage" do
     it "should return an empty array" do
-      file_system_queue.assign_storage({:path => queue_path}).should be_a DispatchRider::QueueServices::FileSystem::Queue
+      expect(file_system_queue.assign_storage({:path => queue_path})).to be_a DispatchRider::QueueServices::FileSystem::Queue
```

Comment:
Line is too long. [125/120]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.